arabic/journey/journey to the unknown.md

```diff
@@ -0,0 +1 @@
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
```

Comment:
Yeah thanks to all my phone was jacked and they took out all my files and put *"-'$*$+. In and I'm lost anyway but they still get in and get everything I do witch ain't much but its still mine and thanks for everyone's help